.github/actions/split-tests/package.json

```diff
@@ -11,6 +11,10 @@
   "keywords": [],
   "author": "",
   "license": "MIT",
+  "engines": {
+    "node": ">=16",
+    "pnpm": ">=8"
```

Comment:
The semver constraints of `^8.0.0` and `^16.0.0` would be better in this case then right?